Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1705231939371)
+++ b/.gitignore	(date 1705231939371)
@@ -0,0 +1,6 @@
+venv/
+.pylintrc
+__pycache__/
+db.sqlite3
+media/
+.idea
\ No newline at end of file
